clustering/web/undertow/src/main/java/org/wildfly/clustering/web/undertow/user/elytron/LocalSSOContext.java

```diff
@@ -3,7 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.wildfly.clustering.web.undertow.sso.elytron;
+package org.wildfly.clustering.web.undertow.user.elytron;
```

Comment:
Unchanged lines cannot be reviewed, but the changes in this commit make this LocalSSOContext classes methods unused.

Reply:
Good catch - this is an orphan class.